.github/workflows/add-tag-to-existing-image.yml

```diff
@@ -0,0 +1,81 @@
+name: Add tag to existing image
+on:
+  workflow_dispatch:
+    inputs:
+      images:
+        type: string
+        description: "Comma separated list of existing images"
+        required: true
+      tags:
+        type: string
+        description: "Comma separated list of tags to add"
+        required: true
+
+env:
+  IMAGES: ${{ inputs.images }}
+  TAGS: ${{ inputs.tags }}
+
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  add-tag-to-existing-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Run add-tag-to-existing-image.sh
+        run: make add-tag-to-existing-image.sh
+      - name: Check for repository changes
+        run: |
+          if git diff --name-only --exit-code; then
+            echo "No changes found in repository"
+            echo "changes_exist=false" >> $GITHUB_ENV
+          else
+            echo "Changes found in repository"
+            git diff --name-only
+            echo "changes_exist=true" >> $GITHUB_ENV
+          fi
+      - name: Create branch, commit and push
+        if: ${{ env.changes_exist == 'true' }}
+        id: branch
+        run: |
+          BRANCH="githubaction-add-tag-$(date +%Y-%m-%d-%H-%M-%S)"
+          echo "branch=${BRANCH}" >> $GITHUB_OUTPUT
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
```

Comment:
Should this come from the environment instead of being hardcoded?

Reply:
The environment contains the actor info that is calling the workflow, these are the values we need to show the commits as coming from the actions actor.

Reply:
Right, but if the environment contains the actor info, couldn't we just pass those env values into the git config, instead of hardcoding them here? I understand that they shouldn't change, but it seems more flexible/reusable to avoid hardcoding things that are controlled by GH and not us.

Reply:
Just because it will never change, I don't see the value in making it configurable. It also introduces a dependency on EIO as now it is controlled outside the team's control (organization variable or per repo which then defeats the purpose of making it configurable)

I would also rather start saving time now by using this then having the perfect implementation, making this configurable is hardly any work and would be included in making it configurable for all repositories as we are already using this configuration in a few repositories (based on how we decide to do it)